e2e_tests/E2ETests.py

```diff
@@ -440,6 +441,7 @@ def cancel_running(self, proc_types: List[const.ProcessType]):
                     "Checkpoint testing: Cancelling job-id {} "
                     "for {} and process type {}".format(entry[1], entry[2], entry[0])
                 )
+                self.canceled_running.append(str(entry[1]))
```

Comment:
Why as str and not int?

Reply:
They get put in an sql query later on, and I think you have to use strings for that

Reply:
Prepared queries (using ?) convert to str for you.